apps/ejabberd/src/mod_mam_iq.erl

```diff
@@ -0,0 +1,140 @@
+-module(mod_mam_iq).
```

Comment:
I really want to name such modules mam_iq, not mod_mam_iq.
Because it's not ejabberd module.

Same with mod_mam_utils. Should be just mam_utils.